test/unit/lib/utils/fs/safeMoveFile.test.js

```diff
@@ -0,0 +1,160 @@
+'use strict';
+
+const fse = require('fs-extra');
+const sinon = require('sinon');
+const chai = require('chai');
+const { getTmpDirPath } = require('../../../../utils/fs');
```

Comment:
Let's rely on `@serverless/test/provision-tmp-dir` instead (note it's _async_).

(we're considering `getTmpDirPath` as deprecated, and it's scheduled to be removed)

Reply:
Oh I see. I've switched that over, Should I use that `@serverless` prefix whenever I'm requiring something, rather than using the relative path?

Reply:
`@serverless/test` simply points to modules in externally maintained [`@serverless/test`](https://www.npmjs.com/package/@serverless/test)

We should use paths naturally (per Node.js rules) if we want to rely on modules hosted in context of this package links should be local, and if we want to import external modules paths should start with names of the packages from which we want import from.